lib/SILGen/SILGenFunction.h

```diff
@@ -466,6 +466,12 @@ class LLVM_LIBRARY_VISIBILITY SILGenFunction
   /// The metatype argument to an allocating constructor, if we're emitting one.
   SILValue AllocatorMetatype;
 
+  /// If set, the the current function an async function which is isolated to
```

Comment:
If set, the current function is an async function which is isolated to

Reply:
It is still `the the`

Reply:
thanks. I tried my best to fix it now :-)